compose/common.yml

```diff
@@ -69,6 +69,13 @@ x-provider-server-environment: &provider-server-environment
   SUBMIT_API_URL: ${SUBMIT_API_URL:-http://cardano-submit-api:8090/}
   USE_BLOCKFROST: ${USE_BLOCKFROST:-false}
   USE_SUBMIT_API: ${USE_SUBMIT_API:-false}
+  ASSET_PROVIDER: ${ASSET_PROVIDER:-dbsync}
```

Comment:
now that we are using microservice asset provider it probably should default to it, not sure if it needs to be changed since API is the same

Reply:
I selected this default based on the default behaviour in the code. 
If `USE_TYPEORM_ASSET_PROVIDER` is true, `ASSET_PROVIDER` selection will be ignored.